src/kokkos_abstraction.hpp

```diff
@@ -486,11 +489,16 @@ struct par_dispatch_impl<Tag, Pattern, Function, TypeList<Bounds...>, TypeList<A
                             Args &&...args, const int scratch_level,
                             const std::size_t scratch_size_in_bytes) {
     static_assert(sizeof...(InnerIs) == 0);
-    kokkos_dispatch(
-        Tag(), name,
-        Kokkos::MDRangePolicy<Kokkos::Rank<Rank>>(exec_space, {bound_arr[OuterIs].s...},
-                                                  {(1 + bound_arr[OuterIs].e)...}),
-        function, std::forward<Args>(args)...);
+    constexpr std::size_t Nouter = sizeof...(OuterIs);
+    Kokkos::Array<int, Nouter> tiling{(OuterIs, 1)...};
```

Comment:
Is this working as expected?
If I infer the intent correctly, this should create an array initialized to `1` everywhere.
My compiler complains with a warning 
```
/p/project/coldcluster/pgrete/athenapk/external/parthenon/src/kokkos_abstraction.hpp(493): warning #174-D: expression has no effect
      Kokkos::Array<int, Nouter> tiling{(OuterIs, 1)...};
```

AFAIK default init doesn't work for arrays, so we might need sth like
```
    std::array<int, Nouter> tiling;
    tiling.fill(1);
```

Reply:
This was working for me, but better to avoid the warning.

The warning makes sense since `(OuterIs, 1)` will always just evaluate to 1.